app/components/sections/users/user-table/component.js

```diff
@@ -0,0 +1,48 @@
+import Ember from "ember";
+import computed from "ember-computed-decorators";
+import UserValidations from "last-strawberry/validators/user";
+import Roles from "last-strawberry/constants/roles";
+
+export default Ember.Component.extend({
+  session: Ember.inject.service(),
+
+  classNames: "col stretch",
+
+  roles: _.values(Roles),
+
+  filterTerm: "",
+
+  @computed("users.@each.{name,email}", "filterTerm")
+  filteredUsers(users, query){
+    return users
+      .filter(user => {
+        const reg = new RegExp(query, "i");
+        return reg.test(user.get("name")) || reg.test(user.get("email"));
+      });
+  },
+
+  @computed("session")
+  validators(session) {
+    return UserValidations(session);
+  },
+
+  actions: {
+    onRequestNewUser() {
```

Comment:
Instead of setting this here, let's add a default role to the user model with pending as default and then just rely on that to set this value